manifests/users.pp

```diff
@@ -56,6 +55,17 @@
     #system         => 'true'
     uid            => '54048',
   }
+  user { 'svna':
+    ensure         => 'present',
+    name           => 'svna',
+    comment        => 'NCSA svna',
+    home           => '/home/svna',
+    managehome     => 'true',
+    password       => '!!',
+    purge_ssh_keys => 'false',
+    shell          => '/bin/bash',
+    system         => 'false',
```

Comment:
Are you sure you don't want this to be a `system` account like most of the other users we are adding? I fear if it isn't there is a chance it collides with an LDAP user UID.

Reply:
Good point. I just grabbed the stanza for `amie` user so just used the same value. I updated `amie` to be `system` as well while I'm at it.

Reply:
I'm not sure changing the Amie user is a good idea. Wouldn't that change the UID and possibly make some data unavailable?

Reply:
From the docs it says flipping `system` won't trigger it to change the uid if the user already exists. Also tested that on `idds-test`  where `svna` was already created as a non system user and it didn't try and change anything.